### PR TITLE
London_10-Saliha_Popal/JavaScript-Core-2-Coursework-Week4

### PR DESCRIPTION
**Pull request by SalihaPopal**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: 1-mandatory/1-alarmclock/alarmclock.js [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+        clearInterval(timerId);
+        playAlarm();
+        flashBgr();
+      }
+      function flashBgr() {
+        var x = Math.floor(Math.random() * 256);
+        var y = Math.floor(Math.random() * 256);
+        var z = Math.floor(Math.random() * 256);
+        var bgColor = "rgb(" + x + "," + y + "," + z + ")";
+     console.log(bgColor);
```
Comment (mira-shukla) [inline 1-mandatory/1-alarmclock/alarmclock.js:24]: Since the values of colours are not changed later is there a different way we can declare this?

File: 1-mandatory/1-alarmclock/index.html [hunk 2]
```diff
@@ -1,5 +1,4 @@
 <!DOCTYPE html>
-<html>
```
Comment (mira-shukla) [inline 1-mandatory/1-alarmclock/index.html:2]: why was this removed?

File: 1-mandatory/2-quotegenerator/quotes.js [hunk 3]
```diff
@@ -490,3 +492,92 @@ const quotes = [
...
+  displayQuote();
+
+
+// function pickFromArray(choices) {
+//   return choices[Math.floor(Math.random() * choices.length)];
+// }
+
+const quoteToggle = document.getElementById("quote-toggle");
+const quoteDisplay = document.getElementById("quote-display");
+const authorDisplay = document.getElementById("author-display");
```
Comment (mira-shukla) [inline 1-mandatory/2-quotegenerator/quotes.js:521]: Really good! just for better readability we could group these DOM elements at the top with the others together

File: 1-mandatory/3-slideshow/slideshow.js [hunk 4]
```diff
@@ -1 +1,97 @@
...
+  clearInterval(intervalId);
+}
+
+function startAutoBackward() { 
+    intervalId = setInterval(moveBackwards, 2000);
+}
+
+function stopAutoBackward() { 
+  clearInterval(intervalId);
+}
```
Comment (mira-shukla) [inline 1-mandatory/3-slideshow/slideshow.js:80]: great structure here!

**Comment by SalihaPopal** [comment]
Hello Mira, thank you for your wise comments. I will definitely use them in
my coding journey.

Best regards,
Saliha
